packages/flutter_test/lib/src/window.dart

```diff
@@ -944,3 +982,263 @@ class TestPlatformDispatcher implements ui.PlatformDispatcher {
     return null;
   }
 }
+
+/// [ui.FlutterView] that wraps another [ui.FlutterView] and allows faking of
+/// some properties for testing purposes.
+///
+/// See also:
+///
+///   * [TestPlatformDispatcher], which allows for faking of platform specific
+///     functionality.
+class TestFlutterView implements ui.FlutterView {
+  /// Constructs a [TestFlutterView] that defers all behavior to the given
+  /// [ui.FlutterView] unless explicitly overridden for testing.
+  TestFlutterView({
+    required ui.FlutterView view,
+    required TestPlatformDispatcher platformDispatcher,
+  }) : _view = view, _platformDispatcher = platformDispatcher {
+    _viewConfiguration = view.viewConfiguration.copyWith(view: this);
```

Comment:
This is actually freezing the view configuration, right? If the configuration on the underlying view were to change, we wouldn't see it in the TestFlutterView. I wonder if that could be a problem? In theory, it shouldn't change while a test is running, but this differs from the original behavior.

Reply:
I'm going to resolve this conversation, as the underlying issue is the same here and for `TestPlatformDispatcher`. It'll be easier to follow in a singular thread.